<ui version="4.0" >
 <class>HubWidget</class>
 <widget class="QWidget" name="HubWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Hub</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="KTextBrowser" name="pChat" >
      <property name="font" >
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTreeView" name="pUsers" >
      <property name="contextMenuPolicy" >
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="pChatInput" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
